test/core/linter/linter_test.py

```diff
@@ -963,59 +501,6 @@ def test__attempt_to_change_templater_warning():
     assert "Attempt to set templater to " in caplog.text
 
 
-@pytest.mark.parametrize(
-    "case",
-    [
-        dict(
-            name="utf8_create",
-            fname="test.sql",
-            encoding="utf-8",
-            existing=None,
-            update="def",
-            expected="def",
-        ),
-        dict(
-            name="utf8_update",
-            fname="test.sql",
-            encoding="utf-8",
-            existing="abc",
-            update="def",
-            expected="def",
-        ),
-        dict(
-            name="utf8_special_char",
-            fname="test.sql",
-            encoding="utf-8",
-            existing="abc",
-            update="→",  # Special utf-8 character
-            expected="→",
-        ),
-        dict(
-            name="incorrect_encoding",
-            fname="test.sql",
-            encoding="Windows-1252",
-            existing="abc",
-            update="→",  # Not valid in Windows-1252
-            expected="abc",  # File should be unchanged
-        ),
-    ],
-    ids=lambda case: case["name"],
-)
-def test_safe_create_replace_file(case, tmp_path):
-    """Test creating or updating .sql files, various content and encoding."""
-    p = tmp_path / case["fname"]
-    if case["existing"]:
-        p.write_text(case["existing"])
-    try:
-        linter.LintedFile._safe_create_replace_file(
-            str(p), str(p), case["update"], case["encoding"]
-        )
-    except:  # noqa: E722
-        pass
-    actual = p.read_text(encoding=case["encoding"])
-    assert case["expected"] == actual
-
-
```

Comment:
Worth noting that this test isn't noqa related, but should really by in `linted_file_test.py` so I've moved it while we're here. It's not been deleted 👍